app/src/main/java/com/example/peterchu/watplanner/Models/Schedule/CourseComponent.java

```diff
@@ -281,15 +281,15 @@ public Calendar getCalendarEndTime() {
 
                     event.setStartTime(eventStartDate);
                     event.setEndTime(eventEndDate);
-                    event.setLocation(location.toString());
+                    event.setLocation(type);
```

Comment:
maybe put the section number up too? so you know which section youre enrolled in